muswarmadmin/repositories.py

```diff
@@ -2,14 +2,47 @@
 import os
 from aiosparql.syntax import IRI, Literal
 from shutil import rmtree
-
 import muswarmadmin.pipelines
 from muswarmadmin.prefixes import Doap, SwarmUI
 
 
 logger = logging.getLogger(__name__)
 
 
+async def repository_has_location(app, pipeline):
+    """
+    Check if the repository associated with the pipeline
+    has a location (git url).
+    """
+    result = await app.sparql.query("""
+        ASK
+        FROM {{graph}}
+        WHERE {
+         ?repository ?p {{pipeline}} .
+         ?repository doap:location ?location .
+        }
+    """, pipeline=pipeline)
```

Comment:
Add one more indent. This is not mandatory but I think this syntax is incorrect. I'm surprised that flake8 doesn't complain.